api/event.ts

```diff
@@ -0,0 +1,21 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import { Group, MeetupEvent } from 'types'
+const groupData = require('../data/data.json')
+
+const GROUP_DATA: Array<any> = (groupData as any).data.groupByUrlname.pastEvents
```

Comment:
hmm this is wrong I think, do not modify this variable, leave it as a group.

also, you should take care of pastEvents and upcomingEvents, because the user can navigate to both kind of events.

I propose you to create a new array of edges from upcoming and past events, in that order, so we first look into upcoming that 99% of the cases will be and empty array, or maybe an array with just or two items, in the worst case, something like:

```
const edges: Edge[] = [...GROUP_DATA.upcomingEvents.edges, ...GROUP_DATA.pastEvents.edges]
```

and perform the find over that array.
does this make sense to you?

Reply:
Yes! my problem was that I couldn't use the `find` method on Group because it wasn't an array, but with this solution is working fine.

Reply:
Why is not good to modify this variable??

Reply:
Hmm that variable serves as a shortcut to get easier the group data and was not intended to be modified... it's not bad per se, I mean we can modify it but in this case you're typing the edge array as any[], and also you're accessing only `pastEvents`...